### PR TITLE
Allow some firmware details as parameters

### PR DESCRIPTION
**Pull request by m-1-k-3**
It is possible to start emba with some additional parameters regarding the firmware under test:

```
Firmware details
-X [version]      Firmware version (double quote your input)
-Y [vendor]       Firmware vendor (double quote your input)
-Z [device]       Device (double quote your input)
-N [notes]        Testing notes (double quote your input)
```

This could result in the following emba startup command:

`sudo ./emba.sh -z -s -l ./logs_dir635 -f /home/m1k3/git-repos/testimages/testimages/testimages/DIR300B5_FW214WWB01.bin -X "2.14-b01" -Y "DLink" -Z "DIR 300 - Wlan router" -N "old firmware" -E`

Emba shows these details on startup:

image

Additionally it gets respect in the aggregator:

image

### CODE REVIEW

File: emba.sh [hunk 1]
```diff
@@ -163,6 +175,32 @@ main()
   done
 
   LOG_DIR="$(abs_path "$LOG_DIR")"
+  print_output "" "no_log"
```
Comment (p4cx) [inline emba.sh:178]: You can use `echo` instead of `print_output "" "no_log`

File: modules/F50_base_aggregator.sh [hunk 2]
```diff
@@ -129,7 +152,11 @@ F50_base_aggregator() {
   fi
 
   if [[ "$STRCPY_CNT" -gt 0 ]]; then
+
     print_output ""
```
Comment (p4cx) [inline modules/F50_base_aggregator.sh:156]: Here you can use `print_output "\\n-----------------------------------------------------------------\\n"`

**Review by p4cx — CHANGES_REQUESTED** [review]

**Comment by p4cx** [comment]
Shellcheck fails here :(

**Comment by m-1-k-3** [comment]
this SC2001 drives me crazy ...

**Review by p4cx — COMMENTED** [review]
Code looks fine to me and will test functionality later.

**Review by p4cx — APPROVED** [review]

**Pull request merged**